prow/setup/ocp_setup.sh

```
@@ -29,13 +29,20 @@ set -x
+ROOT=$(dirname "$WD")
 TIMEOUT=300
 export NAMESPACE="${NAMESPACE:-"istio-system"}"
 export IMAGE_NAMESPACE="${IMAGE_NAMESPACE:-"istio-images"}"
 SAIL_REPO_URL="https://github.com/istio-ecosystem/sail-operator.git"
 SAIL_OPERATOR_BRANCH="${SAIL_OPERATOR_BRANCH:-}"  # Will be auto-detected if not set
 IBM="${IBM:-"false"}"
 
+# Source lib.sh once at the top to make set_topology_value available
+# shellcheck source=prow/lib.sh
+if [ -f "${ROOT}/prow/lib.sh" ]; then
+  source "${ROOT}/prow/lib.sh"
```

Shellcheck (0.11.0)

[info] Not following: prow/lib.sh was not specified as input (see shellcheck -x).

(SC1091)